### PR TITLE
feat: refactor foundation page

### PR DESCRIPTION
**Pull request by PainterPuppets**
reslove: https://github.com/Magickbase/nervos-official-website/issues/439

### CODE REVIEW

**Comment by Keith-CY** [comment]
1. 
image
The image should be scaled in fixed ratio, and the label of the button could be aligned. Maybe the buttons can be stacked vertically if the space is not enough.

2. the cells in background are missing, and the `The Nervos Foundation` in the context should be highlighted
image

3. the navigation links in header are missing
image

4. line-height of context is expected to be larger
image

5. Content of copy is different
image

6. Title should be bold, so do the link, and the arrow icon is differnet from the one in figma
image

7. The gradient color of title is different
image

8. The footer is different
image

**Comment by PainterPuppets** [comment]
> image The image should be scaled in fixed ratio, and the label of the button could be aligned. Maybe the buttons can be stacked vertically if the space is not enough.
> 2. the cells in background are missing, and the `The Nervos Foundation` in the context should be highlighted
> 
> image
> 3. the navigation links in header are missing
> 
> image
> 4. line-height of context is expected to be larger
> 
> image
> 5. Content of copy is different
> 
> image
> 6. Title should be bold, so do the link, and the arrow icon is differnet from the one in figma
> 
> image
> 7. The gradient color of title is different
> 
> image
> 8. The footer is different
> 
> image

updated

**Comment by Keith-CY** [comment]
> > image The image should be scaled in fixed ratio, and the label of the button could be aligned. Maybe the buttons can be stacked vertically if the space is not enough.
> > 2. the cells in background are missing, and the `The Nervos Foundation` in the context should be highlighted
> > image
> > 3. the navigation links in header are missing
> > image
> > 4. line-height of context is expected to be larger
> > image
> > 5. Content of copy is different
> > image
> > 6. Title should be bold, so do the link, and the arrow icon is differnet from the one in figma
> > image
> > 7. The gradient color of title is different
> > image
> > 8. The footer is different
> > image
> 
> updated

LGTM and would be perfect if the link of Nervos DAO leads to an easy-to-understand document instead of a technical one
image

I wouls suggest an article at https://medium.com/nervosnetwork/nervos-dao-explained-95e33898b1c, any idea from @matt-nervos 

The preview is available at https://nervos-official-website-git-fork-painterpuppe-5147e4-magickbase.vercel.app/foundation

**Comment by PainterPuppets** [comment]
The link of the team has been added

File: public/locales/en/foundation.json [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {
   "title": "The Nervos Foundation.",
-  "slogan": "The Nervos Foundation is a non-profit organization whose mission is to support the long-term growth of the Nervos Network. We work to ensure the future success of Nervos by seeding the community to grow independently while driving meaningful innovation in the industry.",
+  "foundation": "Foundation",
+  "slogan": "<tag1>The Nervos Foundation</tag1> is a non-profit organization whose mission is to support the long-term growth of the <tag2>Nervos Network</tag2>. We work to ensure the future success of network by seeding the community to grow independently while driving meaningful innovation in the industry.",
```
Comment (Keith-CY) [inline public/locales/en/foundation.json:4]:
```suggestion
  "slogan": "<tag1>The Nervos Foundation</tag1> is a non-profit organization whose mission is to support the long-term growth of the <tag2>Nervos Network</tag2>. We work to ensure the future success of the network by seeding the community to grow independently while driving meaningful innovation in the industry.",
```

> change "success of Nervos" to "success of the network"

Ref: https://github.com/Magickbase/nervos-official-website/issues/439#issuecomment-2006298757

File: src/pages/foundation/FoundationFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      url: 'https://www.youtube.com/c/NervosNetwork',
+    },
+    {
+      label: <TIcon />,
+      url: 'https://talk.nervos.org/',
+    },
+  ]
+
+  return (
+    <div className={clsx(styles.footer, className)} {...divProps}>
```
Comment (Keith-CY) [inline src/pages/foundation/FoundationFooter.tsx:129]: There is a gap between footer and bottom to make the footer not stick to the edge
image

Others are all good

**Review by Keith-CY — APPROVED** [review]

**Comment by matt-nervos** [comment]
> I wouls suggest an article at https://medium.com/nervosnetwork/nervos-dao-explained-95e33898b1c, any idea from @matt-nervos


I think this article is good @Keith-CY

**Pull request merged**